src/openvic-simulation/map/Province.cpp

```diff
@@ -1,6 +1,7 @@
 #include "Province.hpp"
 
 #include "openvic-simulation/history/ProvinceHistory.hpp"
+#include <algorithm>
```

Comment:
I think this is leftover from when you were using `std::min` here, can you remove it since it's not needed anymore